src/internal/connector/sharepoint/data_collections_test.go

```diff
@@ -41,6 +43,18 @@ type SharePointLibrariesSuite struct {
 	suite.Suite
 }
 
+func (suite *SharePointLibrariesSuite) Client() *msgraphsdk.GraphServiceClient {
```

Comment:
For separation of concerns, the suite itself shouldn't comply with other interfaces.  In the event that we decide to populate these methods for mocking purposes, having them bound to the suite prohibits per-test control of their behavior.  A `type mockServicer struct {}` would be the preferred owner.

Reply:
Resolved within [a1e1ebb](https://github.com/alcionai/corso/pull/2045/commits/a1e1ebb5c32f30458996d1c225ec06fbdab007db)